pkg/signal/trap.go

```diff
@@ -54,11 +53,11 @@ func Trap(cleanup func()) {
 						}
 					} else {
 						// 3 SIGTERM/INT signals received; force exit without cleanup
-						logrus.Info("Forcing docker daemon shutdown without cleanup; 3 interrupts received")
+						fmt.Println("Forcing docker daemon shutdown without cleanup; 3 interrupts received")
 					}
 				case syscall.SIGQUIT:
 					DumpStacks("")
-					logrus.Info("Forcing docker daemon shutdown without cleanup on SIGQUIT")
+					fmt.Println("Forcing docker daemon shutdown without cleanup on SIGQUIT")
```

Comment:
According to  @stevvooe ,  If this is in pkg, it should not be specific to docker/moby and should not import the logger.  I agree with this point.